diffkemp/simpll/CMakeLists.txt

```diff
@@ -39,13 +39,12 @@ if (NOT ${system_libs} STREQUAL "")
     target_link_libraries(simpll ${system_libs})
 endif()
 
-if(SIMPLL_REBUILD_BINDINGS)
-add_custom_target(python-ffi ALL DEPENDS _simpll.c)
-add_custom_command(OUTPUT "${CMAKE_CURRENT_SOURCE_DIR}/_simpll.c"
-                   DEPENDS simpll-lib
-                   WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
-                   COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/simpll_build.py")
-endif()
+add_custom_target(python-ffi ALL DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/_simpll.c")
+add_custom_command(OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/_simpll.c"
+  DEPENDS simpll-lib "simpll_build.py"
+  COMMENT "Building CFFI python bindings for SimpLL"
+  WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
+  COMMAND cp ${CMAKE_CURRENT_SOURCE_DIR}/simpll_build.py . && ./simpll_build.py)
```

Comment:
Just a note: Maybe instead of copying the `simpll_build.py` file it would be possible to use [`tmpdir`](https://cffi.readthedocs.io/en/stable/cdef.html#ffibuilder-compile-etc-compiling-out-of-line-modules) param in `compile` method of `ffibuilder` in `diffkemp/simpll/simpll_build.py` file which should set the directory in which the output files are created.